### DISCUSSION TITLE
How to create an alias of the first header upon note creation?

### DISCUSSION THREAD

**Discussion by JohanHavermans** [SilentVoid13/Templater#436] [Help]
Hi,
I don't know what code has to be in my template to auto create an alias of the first header in a note upon creation.
If auto creation isn't possible, using a shortcut is also fine for me.

Lets assume the filename title of a note is FileName and the content of that note is:
# First Header

bla

What I like to accomplish is that an alias is made of '# First Header' so the file will be:
---
aliases: ["First Header"]
---
# First Header

bla

The idea is that the note is modified upon creation by using Templater, but I can't figure out what code there has to be in the template to be used by Templater. 
I lack the coding experience to see the light here and after a few days I give up finding the solution myself. Would someone please be so kind to help me out?

Thank you in advance!

Johan

**Comment by AB1908** [comment]
I'm late to this but did you figure out out?
